charts/capsule-proxy/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: capsule-proxy
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: 1.16.0
```

Comment:
why `1.16.0` if latest Capsule Proxy is [v0.0.4](https://github.com/clastix/capsule-proxy/releases/tag/v0.0.4)?

Reply:
> we had to rename project and update `Dockerfile`

Renaming has been performed, and keep in mind we provided also a new quay.io repository: `quay.io/clastix/capsule-proxy`, v0.0.5 is already there.

Reply:
fixed